packages/docusaurus-theme-translations/locales/es/theme-common.json

```diff
@@ -51,7 +51,7 @@
   "theme.docs.sidebar.collapseButtonTitle": "Colapsar barra lateral",
   "theme.docs.sidebar.expandButtonAriaLabel": "Expandir barra lateral",
   "theme.docs.sidebar.expandButtonTitle": "Expandir barra lateral",
-  "theme.docs.sidebar.navAriaLabel": "Docs barra lateral",
```

Comment:
Similarly this is an aria label for accessibility

I'm not an expert, but with this change, are you sure a blind person understands this sidebar is related to the docs?

Reply:
thanks for the information i'll updated it based on that